docs/standard/toc.md

```diff
@@ -1,7 +1,7 @@
 # [Get Started with .NET](get-started.md)
 # [Tour of .NET](tour.md)
 # [.NET Architectural Components](components.md)
-# [.NET Standard Library](library.md)
```

Comment:
Yeah, I've been wondering whether I should do that or not. I'll change it to net-standard.